lhotse/audio.py

```diff
@@ -1121,9 +1121,11 @@ def __init__(
             self.reference_energy = audio_energy(base_audio)
         else:
             self.reference_energy = reference_energy
-        assert (
-            self.reference_energy > 0.0
-        ), f"To perform mix, energy must be non-zero and non-negative (got {self.reference_energy})"
+
+        if self.reference_energy <= 0.0:
+            raise NonPositiveEnergyError(
+                f"To perform mix, energy must be non-zero and non-negative (got {self.reference_energy})"
```

Comment:
Can you use the opportunity to display the cut ID?

Reply:
No, the caller needs to do it. The tracks do not contain any cut info.